templates/_partials/header.twig

```diff
@@ -15,7 +15,7 @@
 
 <header class="{{ cx(
   'border-b border-neutral-200 bg-neutral-100',
-  'mdd:sticky mdd:top-0 mdd:z-50 mdd:bg-white mdd:shadow-md',
+  'max-md:sticky max-md:top-0 max-md:z-50 max-md:bg-white max-md:shadow-md',
```

Comment:
I had added some [custom screens config](https://github.com/vigetlabs/craft-site-starter/blob/7e92f31e94cd50d86937b475984a570719471c5c/tailwind.config.js#L24-L39) to create `max` media queries. 

But this is now part of TW 4 (or might have [been since 3 actually](https://v3.tailwindcss.com/docs/responsive-design))